app/components/sy-topnav/component.js

```diff
@@ -12,4 +12,23 @@ export default class SyTopnav extends Component {
   get navMobile() {
     return this.media.isMo || this.media.isXs || this.media.isSm;
   }
+
+  /**
+   * TODO:
+   * i don't know if it's an issue in `ember-simple-auth` or `ember-simple-auth-oidc`
+   * or it's maybe wrong configration from Timed frontend side,
+   * BUT after refreshing the token, the session service is not encapsulating the reslts from keycloak
+   * inside ember data model, so i add this small.
+   *
+   * if the issue is fixed, change everything in app/abilities/report.js that belongs to `this.user?.id`
+   */
```

Comment:
because in `app/abilities/report.js`, i remove `.get` so if the issue is fixed, it will be better to return it again to use `.get` to access the attribute

Reply:
Shouldnt get anways get removed? In newer ember-data versions get is not necessary to retrieve attributes which are not relationships or is user not from ember-data.

Reply:
then, i will remove the comment that belong to replacing report abilities part